### PR TITLE
Update uninstall stanza for folding-at-home

### PR DESCRIPTION
**Pull request by unitof**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

### CODE REVIEW

File: Casks/folding-at-home.rb [hunk 1]
```diff
@@ -16,4 +16,8 @@
               "org.foldingathome.fahviewer",
               "org.foldingathome.fahcontrol",
             ]
+
+  zap trash: [
+    "/Library/Application Support/FAHClient",
```
Comment (miccal) [inline Casks/folding-at-home.rb:21]: Just confirming that these files are in `/Library`, and not  `~/Library`?

Comment (unitof) [reply Casks/folding-at-home.rb:21]: Yep, confirmed.
image

Comment (miccal) [reply Casks/folding-at-home.rb]: Thanks @unitof -- that means that they are not user-specific files, and belong in `uninstall` not `zap`.

Comment (unitof) [reply Casks/folding-at-home.rb]: Gotcha, makes sense!

**Comment by miccal** [comment]
Thank you @unitof.

**Review by miccal — APPROVED** [review]

**Comment by unitof** [comment]
Hm, it seems to only remove the `FAHClient` directory (and any subdirectories) if empty, which is not desired.

Does `rmdir` by default only delete empty directories?

```
==> Uninstalling packages:
==> Removing directories if empty:
/Library/Application Support/FAHClient
/Library/Application Support/FAHClient/logs
/Library/Application Support/FAHClient/work
/Library/Application Support/FAHClient/work/00
/Library/Application Support/FAHClient/work/00/01
/Library/Application Support/FAHClient/cores
/Library/Application Support/FAHClient/cores/cores.foldingathome.org
/Library/Application Support/FAHClient/cores/cores.foldingathome.org/osx
/Library/Application Support/FAHClient/cores/cores.foldingathome.org/osx/64bit-sse2
/Library/Application Support/FAHClient/cores/cores.foldingathome.org/osx/64bit-sse2/a8-0.0.9
/Library/Application Support/FAHClient/cores/cores.foldingathome.org/osx/64bit-sse2/a8-0.0.9/Core_a8.fah
/Library/Application Support/FAHClient/cores/cores.foldingathome.org/osx/64bit-sse2/a7-0.0.19
/Library/Application Support/FAHClient/cores/cores.foldingathome.org/osx/64bit-sse2/a7-0.0.19/Core_a7.fah
```

**Comment by miccal** [comment]
> Does rmdir by default only delete empty directories?

Yes it does -- do you thinks these should be removed and re-created when this is upgraded and/or reinstalled.

**Comment by unitof** [comment]
Probably not. But it's definitely an artifact left by `folding-at-home` installation which is not removed by `brew uninstall --cask folding-at-home` currently. Perhaps it should move back to the `zap` stanza?

**Comment by miccal** [comment]
Ping @vitorgalvao for his thoughts.

**Comment by vitorgalvao** [comment]
> Does `rmdir` by default only delete empty directories?

Not “by default”, but at all. [That’s how `rmdir` is supposed to work](https://en.wikipedia.org/wiki/Rmdir).

Do these directories (being in `/Library` as opposed to `~/Library`) ever store user data or settings? I.e., do its contents change from usage? If not, they should be in `uninstall`, otherwise `zap`. But it’s not a big issue if they’re in `zap`.

**Comment by unitof** [comment]
I'm not sure exactly what's stored in the `/Library/.../FAHClient` directory. I'll dig around after running it next time, but my guess is it's not user data.

**Pull request merged**